### ISSUE TITLE
Game window freezes a couple of seconds after connecting.

### ISSUE THREAD

**Issue by kol1th** [d3athrow/vgstation13#13241]
I've tried restarting, starting byond as administrator, and updating to latest byond beta. 

The game window just freezes a second or two after connecting, and I can't find why, what causes it, or how to fix it.

**Comment by Probe1** [comment]
https://github.com/d3athrow/vgstation13/issues/13240

**Comment by kol1th** [comment]
It's not a duplicate of that issue. In fact, it's the opposite. The *game window* won't respond. It just freezes, but the rest works just fine.

**Comment by Probe1** [comment]
Navigate to Byond->Preferences->Games->Delete cache.

**Comment by kol1th** [comment]
Cleared cache, still fucked

**Comment by Ephx1** [comment]
Question: Are you on beta version? I had something similar happen, and downgrading to the regular version helped. If you have the same problem as I, switching back and forth between text and graphic mode should update the game screen each time.

**Comment by kol1th** [comment]
Switching to stable will break other things. and also doesn't fix it. Doesn't happen every round either. This was actually the first time it happened, but I have had cases of it happening to others under the same circumstances.

**Comment by Ephx1** [comment]
Yeah, it didn't happen every round but would occasionally just freeze and reconnecting would have it either display nothing or freeze after a few seconds. Weird, on stable I didn't have any issues.
Might I ask what it's supposed to break?

**Comment by pvak** [comment]
i had this similar issue, everything was fine except for the game screen to be frozen and reconnecting would turn the screen black while rejoing the server would unfreeze it for like 2 seconds

**Comment by BobdaBiscuit** [comment]
Rclick the title bar and go to options and messages, it could be the game is just downloading minimaps etc.

**Comment by kol1th** [comment]
Still happening. Options and Messages isn't downloading shit. It only happens on /vg/, and only after I reconnect once. Don't know if this will help. 

![image](https://cloud.githubusercontent.com/assets/16979948/22009100/ac91d416-dc35-11e6-8a2a-c74922eeb297.png)
